x/distribution/client/cli/query.go

```diff
@@ -181,9 +181,43 @@ func GetCmdQueryDelegatorRewards(queryRoute string, cdc *codec.Codec) *cobra.Com
 				return err
 			}
 
-			var coins sdk.DecCoins
-			cdc.MustUnmarshalJSON(res, &coins)
-			return cliCtx.PrintOutput(coins)
+			var resp distr.DelegationRewards
+			cdc.MustUnmarshalJSON(res, &resp)
+			return cliCtx.PrintOutput(resp)
+		},
+	}
+}
+
+// GetCmdQueryAllDelegatorRewards implements the query all delegator rewards
+// command.
+func GetCmdQueryAllDelegatorRewards(queryRoute string, cdc *codec.Codec) *cobra.Command {
```

Comment:
I'm thinking we can consolidate this into the `rewards` command and make the validator address optional. Thoughts @cwgoes?

Reply:
Sure.